x-pack/solutions/security/plugins/security_solution/public/assistant/provider.tsx

```diff
@@ -234,6 +236,7 @@ export const AssistantProvider: FC<PropsWithChildren<unknown>> = ({ children })
       inferenceEnabled={inferenceEnabled}
       navigateToApp={navigateToApp}
       productDocBase={productDocBase}
+      spaceId={spaceId ?? 'default'}
```

Comment:
Should we consider only render `ElasticAssistantProvider` when spaceId becomes available? Fallback to `default` might causes incorrect local storage key although chances are small.